src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -1047,7 +1047,7 @@ class MessageSection extends Component {
    }
 
   componentDidUpdate() {
-    switch (this.state.currTheme) {
+      switch (this.state.currTheme) {
```

Comment:
Why this change?

Reply:
I think these changes are shown because of rebase.